examples/customize-toolbar-excel-download/index.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import MUIDataTable from '../../src';
+import CustomToolbar from './CustomToolbar';
+// import XLSX from 'xlsx';
+
+const Example = () => {
+  const [tableState, setTableState] = React.useState();
```

Comment:
Can we avoid using hooks, as this is a newer React feature that won't work on lower versions, such as those supported by this library (which will also make this example more inclusive, since more people will be able to run it).

Reply:
Sure thing. I'm very deep into TS and Hooks these days but I'll fix it up today.